### ISSUE TITLE
Error regarding BDSHOT on Aux1

### ISSUE THREAD

**Issue by joe-stroup** [ArduPilot/ardupilot_wiki#5133]
There is an error in the wiki regarding the use of AUX1 for bi-directional DSHOT.

I've had passthrough work on AUX-1 OK on some ESC's, but fail on others.  However, this does not effect using AUX for BDSHOT - you just have to move the ESC to another AUX port to run pass through for BLHeliSuite32 to configure the ESC.

Also - BDSHOT requires AUX ports to be used in certain groupings.  So the wording of the wiki gives the impression that you could use AUX 2-5 - and as I under stand it, this grouping won't work.

A good resource to verify these details is Andy Piper.

![wiki error bdshot aux1](https://github.com/ArduPilot/ardupilot_wiki/assets/91474707/0d060d05-07a4-4d5b-ac33-02c58437ab2f)

![aux1 showing not working after loading bdshot firmware on holybrow tekko32](https://github.com/ArduPilot/ardupilot_wiki/assets/91474707/9903dc33-ca21-4c3b-983e-d2630a3569a7)

![passthrough working on aux1](https://github.com/ArduPilot/ardupilot_wiki/assets/91474707/fc346602-f2fe-4c67-a20c-52a26eeb7ab5)

**Comment by Hwurzburg** [comment]
@andyp1per  pls confirm and suggest required changes

**Comment by andyp1per** [comment]
Yeah I think this is correct, the general problem with AUX1 is that serial passthrough does not work. BDSHOT, however, should work. So just replace "bi-directional dshot" with "serial passthrough"

**Comment by Hwurzburg** [comment]
The third paragraph after this links directly to section on grouping..so it's mentioned directly on this page...
